### ISSUE TITLE
Purging scanned sheets

### ISSUE THREAD

**Issue by ysalmon** [sdaps/sdaps#188]
What is a correct way to purge the scanned answer sheets of a survey (in case something goes wrong during recognition, I prefer to throw everything out and re-recognize everything) ?

With earlier version of sdaps, I would delete the entire project and regenerate it.
But with the database now being in sqlite format, I feel tempted to just ``DELETE FROM sheets;`` and delete the 1.tif, 2.tif, etc. files.
Is this safe ?

**Comment by benzea** [comment]
Yeah, should be safe. There is also a `reset` command to delete everything, not sure if that currently works though.

**Comment by benzea** [comment]
> The JSON in the surveys table references the project folder : what happens if this does not correspond to the actual folder anymore?

Wait, huh? That shouldn't be the case. The `survey_dir` attribute should not be saved (and is never used).
